agent/unit-test/configstore_unittest.cpp

```diff
@@ -30,89 +31,165 @@
 
 using namespace softwarecontainer;
 
+/*
+ * Test stub - ServiceManifestStringLoader
+ *
+ * Loads a service manifests from a string, as opposed to the non-test loader
+ * which reads from file.
+ */
+class ServiceManifestStringLoader : public ServiceManifestLoader
+{
+LOG_DECLARE_CLASS_CONTEXT("SMSL", "SoftwareContainer service manifest loader");
+
+public:
+    // Constructor needs to init parent with the service manifest source string
+    ServiceManifestStringLoader(const std::string &source) :
+        ServiceManifestLoader(source)
+    {}
+    ~ServiceManifestStringLoader()
+    {
+        for (json_t *item : m_content) {
+            json_decref(item);
```

Comment:
You are right. I moved it to the base class and forgot to remove it here. I will remove this.